pkgs/build-support/rust/default.nix

```diff
@@ -32,6 +32,11 @@
 , depsExtraArgs ? {}
 , cargoParallelTestThreads ? true
 
+# If set the CARGO_TARGET_DIR will be set to this directory. This will allow
+# you to avoid rebuilding dependencies.
+# This setting is *incompatible* with sandboxing.
+, impureCacheDir ? null
```

Comment:
This also should be in the manual, ideally with an example. I assume one could use a relaxed sandbox, where only the rust package is built impure (`__noChroot = true;`)

Reply:
I checked the docs and there isn't an obvious place to put this as there is no attribute reference. Is there a spot that you would recomend?